vms/registration/forms.py

```diff
@@ -5,8 +5,16 @@
 
 class UserForm(forms.ModelForm):
     # password not visible when user types it out
-    password = forms.CharField(widget=forms.PasswordInput())
+    password = forms.CharField(widget=forms.PasswordInput(), min_length=8, max_length=25)
```

Comment:
Any reason for 25 max length? Usually its best practice for maximum length of password field to be of 128 characters.